egs/csj/s5/local/csj_make_trans/vocab2dic.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#! /usr/bin/perl
```

Comment:
use /usr/bin/env perl.
you can use "use Warnings" within the script to enable the warnings